webCommander/centralHub/spotify/spotifyAPI.py

```diff
@@ -48,35 +51,56 @@ def get_spotify_artist(artist_name):
     Authenticate then search an artist based on an artist name. Searches for the name if id is not provided
     '''
 
-    if len(artist_name) != 22:
-        artist_id = search_spotify_artist(artist_name)
+    #TODO 
+    #Spotify artist is a 22 fixed character ID, but searching for a word would require to first call search_spotify_artist 
```

Comment:
is this still needed after our call? I mean I saw type=artist in the api response or was that for another call?

Reply:
done